src/lxc/cgroups/cgroup.h

```diff
@@ -146,7 +153,7 @@ struct cgroup_ops {
 	bool (*monitor_enter)(struct cgroup_ops *ops, struct lxc_handler *handler);
 	bool (*payload_create)(struct cgroup_ops *ops, struct lxc_handler *handler);
 	bool (*payload_enter)(struct cgroup_ops *ops, struct lxc_handler *handler);
-	const char *(*get_cgroup)(struct cgroup_ops *ops, const char *controller);
+	const char *(*get_cgroup)(struct cgroup_ops *ops, const char *controller, bool limiting);
```

Comment:
Hm, wouldn't a new `get_limiting_cgroup()` be nicer and move that abstraction with the `bool` argument into `cgfsng.c`?

Reply:
I think that would make the codepath in `commands.c` a little nicer, too.

Reply:
True, will change it. (At least this prevented me from forgetting to handle all the call sites :) )